content/developer/reference/standard_modules/account/account_chart_template.rst

```diff
@@ -0,0 +1,62 @@
+.. _reference/account_chart_template:
+
+==============
+Chart Template
+==============
+
+---------------------
+Invoicing (`account`)
+---------------------
+
+This is the main module for Invoicing.
+
+.. autoclass:: odoo.addons.account.models.chart_template.AccountChartTemplate
+
+    .. autofield:: name
+    .. autofield:: parent_id
+    .. autofield:: code_digits
+    .. autofield:: visible
+    .. autofield:: currency_id
+    .. autofield:: country_id
+    .. autofield:: use_anglo_saxon
+    .. autofield:: bank_account_code_prefix
+    .. autofield:: cash_account_code_prefix
+    .. autofield:: transfer_account_code_prefix
+    .. autofield:: account_ids
+    .. autofield:: income_currency_exchange_account_id
+    .. autofield:: expense_currency_exchange_account_id
+    .. autofield:: account_journal_suspense_account_id
+    .. autofield:: account_journal_payment_debit_account_id
+    .. autofield:: account_journal_payment_credit_account_id
```

Comment:
The same about suspense accounts. Don't make conspiracy about it. People should know the truth

# Current version
![image](https://user-images.githubusercontent.com/186131/143898868-805eda3b-ecab-4727-af36-59127a8a5d66.png)

# My version

![image](https://user-images.githubusercontent.com/186131/143899238-4c20d0ab-4361-414b-b662-6b59afbfc467.png)

Reply:
We will create a PR that adds `help` on the fields. It wouldn't be synchronized anyway.

Reply:
Why it won't be synchronized ? You plan to add help to master branch, right?
We either need such PR to odoo/odoo v15 or keep fields description in odoo/documentation v15 for a while (and replace it  with generated docs in v16). I don't like the idea to hide the knowledge and enlighten new developers during special ceremony with accounting gurus.

Reply:
It isn't synchronized because runbot team says it is too difficult to do to have a batch containing both the code and the doc